unlock-app/src/hooks/useLockSettings.ts

```diff
@@ -70,3 +77,25 @@ export function useSaveSlugSetting() {
     }
   })
 }
+
+export function useSaveLockSettings() {
+  return useMutation(async (settings: SettingsProps) => {
+    const { lockAddress, network } = settings
+
+    return storage.saveLockSetting(network, lockAddress, {
+      ...settings,
+    })
+  })
+}
+
+export function useGetLockSettings({
```

Comment:
Same here.
Also, let's actually move that to a different PR please.

Reply:
i will move in a different PR